injected/src/features/autofill-password-import.js

```diff
@@ -157,17 +258,27 @@ export default class AutofillPasswordImport extends ContentFeature {
      * @param {string} path
      */
     async handleElementForPath (path) {
+        this.removeOverlayIfNeeded()
         const supportedPaths = [
             this.#exportButtonSettings?.path,
             this.#settingsButtonSettings?.path,
             this.#signInButtonSettings?.path
         ]
-        if (supportedPaths.indexOf(path)) {
+        if (supportedPaths.includes(path)) {
             try {
-                const { element, style, shouldTap } = await this.getElementAndStyleFromPath(path) ?? {}
+                const { element, style, shouldTap, shouldWatchForRemoval } = await this.getElementAndStyleFromPath(path) ?? {}
                 if (element != null) {
-                    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
-                    shouldTap ? this.autotapElement(element) : this.animateElement(element, style)
+                    if (shouldTap) {
+                        this.autotapElement(element)
+                    } else {
+                        setTimeout(() => this.animateElement(element, style), ANIMATION_TIMEOUT)
```

Comment:
Maybe this is safer:

```js
const domLoaded = new Promise((resolve) => {
  if (document.readyState === "loading") {
    document.addEventListener("DOMContentLoaded", resolve);
  } else {
    resolve();
  }
})
```

```suggestion
                        await domLoaded
                        this.animateElement(element, style)
```